docs/usage/configuration-options.md

```diff
@@ -3539,6 +3539,12 @@ Configure this to `true` if you wish to get one PR for every separate major vers
 e.g. if you are on webpack@v1 currently then default behavior is a PR for upgrading to webpack@v3 and not for webpack@v2.
 If this setting is true then you would get one PR for webpack@v2 and one for webpack@v3.
 
+## separateMultipleMinor
+
+Configure this to `true` if you wish to get one PR for every separate minor version upgrade of a dependency.
```

Comment:
the description from the config option will be added here too, so it seems this sentence is duplicate?